.github/workflows/npm.yml

```
@@ -0,0 +1,36 @@
+name: NPM
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "contracts/**"
+      - "package.json"
+      - "package-lock.json"
```

We're using yarn in this project.
[yarn.lock](https://github.com/thesis/solidity-contracts/blob/main/yarn.lock)
```suggestion
      - "yarn.lock"
```

---

See 42e9809ff0c649a066cc53fdbe881e810e0e15c8